cosmoz-viewinfo-mixin.js

```diff
@@ -0,0 +1,70 @@
+import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin';
+
+export const VIEW_INFO_INSTANCES = [],
+	SHARED_VIEW_INFO = {
+		desktop: false,
+		effects: 10,
+		height: 0,
+		landscape: false,
+		mobile: false,
+		portrait: true,
+		tablet: false,
+		width: 0
+	};
+
+/**
+ * Mixin to inherit the viewInfo property with device and viewport info.
+ *
+ * @polymer
+ * @mixinFunction
+ */
+export const ViewInfoMixin = dedupingMixin((base)=>
```

Comment:
```suggestion
export const ViewInfoMixin = dedupingMixin(base => class extends base {
```
is more in line with guidelines (actually I think the current will error with up to date eslint rules) and allow one less level of indentation.